x-pack/plugins/security/server/authorization/disable_ui_capabilities.ts

```diff
@@ -75,11 +75,15 @@ export function disableUICapabilitiesFactory(
   const shouldAffectCapability = (featureId: keyof UICapabilities, uiCapability: string) => {
     // This method answers: 'Should we affect a capability based on privileges?'
 
-    // 'spaces' and 'fileUpload' feature ID's are handled independently
-    // The spaces and file_upload plugins have their own capabilites switchers
+    // 'fileUpload' feature ID is handled independently
+    // The spaces and file_upload plugins have their own capabilites switchers.
```

Comment:
__typo:__ (it was like that before though)
```suggestion
    // 'fileUpload' feature ID is handled independently
    // The spaces and file_upload plugins have their own capabilities switchers.
```